packages/tree-widget/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@bentley/tree-widget-react",
+  "version": "2.0.0",
```

Comment:
any reason to start at `2.0.0` instead of `1.0.0`?

Reply:
none at all. will change to 1.0.0

Reply:
fixed